params/config.go

```diff
@@ -813,6 +814,7 @@ type Rules struct {
 	IsHomestead, IsEIP150, IsEIP155, IsEIP158               bool
 	IsByzantium, IsConstantinople, IsPetersburg, IsIstanbul bool
 	IsYoloV1                                                bool
+	IsPrivacyEnhancementsEnabled                            bool
```

Comment:
`Rules` struct has been added in this Geth version?

Reply:
no, it was already there. I think you looked to the commit that added Quorum into this geth branch and no the other way around. 

the only thing that added (if you check latest version) is `IsYoloV1                                                bool`